content/en/blog/recap-2022/index.md

```diff
@@ -0,0 +1,137 @@
+
+- Falcosidekick saw a major update in June of 2022, delivering four new outputs and enhancing existing ones too! Not only did we see updates in outputs, but the community delivered the first version of the Falcosidekick UI. The UI offered these new features: Redis DB for long term storage of events, API for counting and searching for events, and lastly filters to keep and share query strings. For a full detailed account of this update, see the blog Falcosidekick 2.25.0 and Falcosidekick 2.0.0. 
+
+- Well, we thought the June release was big, but Thomas had some goodies for the community just in time for Christmas! The number of new outputs for this release include: Yandex, MQTT & Node-Red, plus many more totaling to eight new outputs! However, it’s not just new outputs that were added but also enhancements to the existing outputs were included in this release. The new features for Falcosidekick 2.27.0 also include: support for both SMTP & Kafka, which can now benefit from SASL auth mechanisms, now you can use environment variables to customize your fields, create policy reports, among a few other very cool features. To read the full release notes, take a look at  [Falcosidekick 2.27.0 and Falcosidekick-UI 2.1.0](https://falco.org/blog/falcosidekick-2-27-0-ui-2-1-0/).
+
+### Plugins 
+Falco v0.31.0 resulted in many exciting new features. One that is particularly strategic for the project is the general availability of the plugins framework. Why are plugins exciting and what do they mean for the future of Falco? 
+
+Plugins are shared libraries that can be loaded by Falco to extend its functionality. Plugins come in two flavors:
+
+- Source Plugins, which add new data sources to Falco. They produce input events, from either the local machine or a remote source, that Falco can understand.
+- Extractor Plugins, which parse the data coming from source plugins and expose new fields that can be used in Falco rules.
+
+The combination of Source and Extractor plugins allows users to feed data into Falco, parse it in useful ways and create rules and policies from it, read the full announcement [here!](https://falco.org/blog/falco-announcing-plugins/) 
+
+Official plugins launched in 2022 include: 
+- AWS CloudTrail
+- GitHub
+- K8saudit
+- K8saudit-EKS
+- Okta
+
+### Falco project growth
+The Falco project excitedly announced that on Nov 4, 2022, project maintainers submitted the proposal to be considered for graduation. This milestone is important because it demonstrates the health and maturity of the project to the Kubernetes & cloud ecosystem.
+
+Falco joined the CNCF as a Sandbox project in the fall of 2018 and evolved into Incubating status two years later. The community believes the next natural step is for Falco to apply for graduation. The project has reached maturity in community health, diversity of contributions & contributors, and ecosystem adoption.
+
+- Project highlights for 2022
```

Comment:
How are these calculated? For example, looking at [Falco's devstats](https://falco.devstats.cncf.io/d/3/stars-and-forks-by-repository?orgId=1&from=1638313200000&to=1669849200000) the growth in stars and forks across all repos are ~+31% and ~+46% since last year.

Reply:
took these from the dd doc